### PR TITLE
French translation pass

### PR DESCRIPTION
**Pull request by alganthe**
some corrections and the medical stringtable completed:.

Be advised that i'm no medic and that some medical words "may" be wrong
in the context they are used, especially for velocity wounds. for the
rest it should be fine.

other changes:
claymore name changed, was too long
Sanitaire changed to Médical for trucks.

Some translation on missile guidance.

### CODE REVIEW

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -1406,23 +1427,28 @@
...
             <English>He's lost some blood</English>
+            <French>Il a perdu du sang</French>
         </Key>
         <Key ID="STR_ACE_MEDICAL_noBloodloss">
             <English>He hasn't lost blood</English>
+            <French>Il n'a pas perdu de sang</French>
         </Key>
         <Key ID="STR_ACE_MEDICAL_inPain">
             <English>He is in pain</English>
+            <French>Ill souffre</French>
```
Comment (bux578) [inline addons/medical/stringtable.xml]: `Il souffre`

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -1754,142 +1780,167 @@
         <Key ID="STR_ACE_Medical_Wounds_Abrasion">
             <English>Scrape</English>
             <German>Kratzer</German>
+            <French>éraflure</French>
```
Comment (bux578) [inline addons/medical/stringtable.xml]: `Éraflure`

File: addons/medical/stringtable.xml [hunk 3]
```diff
@@ -1754,142 +1780,167 @@
...
-        
         <Key ID="STR_ACE_Medical_Wounds_Contusion">
             <English>Bruise</English>
             <German>Prellung</German>
+            <French>Hématome</French>
         </Key>
         <Key ID="STR_ACE_Medical_Wounds_Contusion_Minor">
             <English>Minor Bruise</English>
             <German>Kleine Prellung</German>
+            <French>Hématome mineur</French>
```
Comment (bux578) [inline addons/medical/stringtable.xml]: `Mineur` comme `Avulsion Mineure` ?

File: addons/medical/stringtable.xml [hunk 4]
```diff
@@ -1754,142 +1780,167 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_Wounds_Cut_Minor">
             <English>Small Cut</English>
             <German>Kleine Schnittwunde</German>
+            <French>Petite coupure</French>
         </Key>
         <Key ID="STR_ACE_Medical_Wounds_Cut_Medium">
             <English>Medium Cut</English>
             <German>Mittlere Schnittwunde</German>
+            <French>Moyenne Coupure</French>
```
Comment (bux578) [inline addons/medical/stringtable.xml]: `Moyenne Coupure` vs `Petite coupure`

File: addons/medical/stringtable.xml [hunk 5]
```diff
@@ -1754,142 +1780,167 @@
...
         </Key>
         <Key ID="STR_ACE_Medical_Wounds_VelocityWound_Medium">
             <English>Medium Velocity Wound</English>
             <German>Mittleres Ballistisches Trauma</German>
+            <French>Moyenne Blessure de vélocité</French>
         </Key>
         <Key ID="STR_ACE_Medical_Wounds_VelocityWound_Large">
             <English>Large Velocity Wound</English>
             <German>Großes Ballistisches Trauma</German>
+            <French>Large blessure de vélocité</French>
```
Comment (bux578) [inline addons/medical/stringtable.xml]: `blessure` vs `Blessure`

File: addons/mk6mortar/stringtable.xml [hunk 6]
```diff
@@ -1,17 +1,20 @@
...
+        <Key ID="STR_ACE_MK6MORTAR_rangetable_description">
+            <English>Range Table for the MK6 82mm Mortar</English>
+            <French>Table de tir pour le mortier MK6 82mm</French>
+        </Key>
+        <Key ID="STR_ACE_MK6MORTAR_rangetable_action">
+            <English>Open 82mm Rangetable</English>
+            <French>ouvrir la table de tir pour 82mm</French>
+        </Key>
+        <Key ID="STR_ACE_MK6MORTAR_rangetable_charge">
+            <English>Charge</English>
```
Comment (bux578) [inline addons/mk6mortar/stringtable.xml]: oublié?

File: addons/realisticnames/stringtable.xml [hunk 7]
```diff
@@ -867,7 +867,7 @@
             <Spanish>M18A1 Claymore</Spanish>
             <Polish>Mina kierunkowa M18A1 Claymore</Polish>
             <Czech>Mina Claymore M18A1</Czech>
-            <French>M18A1 Claymore Mine antipersonnel à effet dirigé</French>
+            <French>M18A1 Claymore </French>
```
Comment (bux578) [inline addons/realisticnames/stringtable.xml]: Espace.

File: addons/medical/stringtable.xml (hunk 3 above)
Comment (alganthe) [inline addons/medical/stringtable.xml]: Hématome est masculin dans ce cas. C'est donc un Hématome mineur

File: addons/mk6mortar/stringtable.xml (hunk 6 above)
Comment (alganthe) [inline addons/mk6mortar/stringtable.xml]: Pour charge ? non je ne comprends just pas à quoi cela fait référence.

File: addons/medical/stringtable.xml (hunk 3 above)
Comment (bux578) [inline addons/medical/stringtable.xml]: `M` vs `m`

File: addons/mk6mortar/stringtable.xml (hunk 6 above)
Comment (bux578) [inline addons/mk6mortar/stringtable.xml]: Top left:
https://cloud.githubusercontent.com/assets/9376747/7001355/ba70c6a4-dbfc-11e4-8d59-8a71e469fcde.jpg

**Comment by alganthe** [comment]
everything fixed, I still don't get what the charge actually mean in that case so i'll leave it aside, i'll do a second pass tomorrow to check what's left.

**Comment by ulteq** [comment]
>I still don't get what the charge actually mean in that case

http://www.inetres.com/gp/military/infantry/mortar/81mm/81_M374A3.gif

**Comment by alganthe** [comment]
thanks to ulteq I know what the charge is, I fucked up since I didn't knew you couldn't rebase a forked repo I'll close this issue and re-send a proper pull request later with a bit more adds.

**Pull request closed without merging**